### DISCUSSION TITLE
Domain labels get cut off if too long

### DISCUSSION THREAD

**Discussion by Emporea** [observablehq/plot#718] [Q&A] [answered]
Whats the proper way to handle cut off domain labels?
![image](https://user-images.githubusercontent.com/21969523/151586655-6c51cfc9-ce53-4659-b8c7-4355e6bbbecd.png)

For example changing the x,y of viewBox to -20 -20 doesnt help. Its still cut off.

**Comment by mbostock** [comment, marked as answer]
The marginLeft option.

**Reply by Emporea** [reply]
Awesome!
If I want it to change automatically I guess I have to calculate d3.max of longest string * constant ?

**Reply by mbostock** [reply]
Yep. Related https://github.com/observablehq/plot/issues/394 https://github.com/observablehq/plot/issues/383
